src/adapters/destination-adapters/ChronicleMedianDestinationAdapter.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.17;
+
+import {IMedian} from "../../interfaces/chronicle/IMedian.sol";
+import {DiamondRootOval} from "../../DiamondRootOval.sol";
+
+/**
+ * @notice ChronicleMedianDestinationAdapter contract to expose Oval data via the standard Chronicle interface.
+ */
+
```

Comment:
nit: `forge fmt` would remove empty line here and everywhere else after the contract NatSpec